webxr-api/registry.rs

```diff
@@ -234,7 +238,12 @@ where
 
 #[cfg_attr(feature = "ipc", derive(Serialize, Deserialize))]
 enum RegistryMsg {
-    RequestSession(SessionMode, Sender<Result<Session, Error>>, Sender<Frame>),
+    RequestSession(
+        SessionMode,
+        SessionInit,
```

Comment:
In https://immersive-web.github.io/webxr/#dom-xr-requestsession the options are optional.

Reply:
That's a quirk of webidl, dictionary arguments with all optional fields must be optional. The bindings generation already takes care of this for us.